Snakefile

```diff
@@ -151,8 +149,8 @@ rule download_metadata:
 
 rule filter_sequences_by_subtype:
     input:
-        sequences="data/{segment}/sequences.fasta",
-        metadata="data/metadata.tsv",
+        sequences=lambda w: "ingest/results/sequences_{segment}.fasta" if LOCAL_INGEST else "data/{segment}/sequences.fasta",
+        metadata=lambda w: "ingest/results/metadata.tsv" if LOCAL_INGEST else "data/metadata.tsv",

```

Comment:
> I'd prefer to avoid additional conditional inputs in the workflow and stick with the norm from the example data usage in this repo's README and [the pathogen-repo-guide](https://github.com/nextstrain/pathogen-repo-guide/blob/6e9e77b94cd7688b4e8a35615a1a06796a4fcf9a/phylogenetic/rules/prepare_sequences.smk#L4-L8) where the workflow requires the initial data to be in data/ and the user needs to copy those data into the correct folder to start... If I have to make that choice as a config parameter or a cp command, I might as well do the cp command.

I see this as analogous to defining a different `s3_src`. Furthermore a simple `cp` command is complicated by the naming conventions chosen in this repo where we have `ingest/results/sequences_{SEGMENT}.fasta` but `data/{SEGMENT}/sequences.fasta` -- we can construct the appropriate `cp` command, but I find it better communicated through snakemake. 

I've changed the snakemake style in [36c4b6a](https://github.com/nextstrain/avian-flu/pull/27/commits/36c4b6aa0c3874287eaccee25988c7824715bdbc) - do you find this any better?

Reply:
> Mostly because we don't use this approach anywhere else

Thinking about this more on my bike ride home, I realize this isn't constructive point of view if you're suggesting that we _should_ use this approach throughout our workflows as a general way of simplifying the ingest-to-phylogenetic transition for users with their own local data. Maybe worth a chat this week sometime?

Reply:
> if you're suggesting that we should use this approach throughout our workflows as a general way of simplifying the ingest-to-phylogenetic transition for users with their own local data

Yeah, from the start I've been trying to push for "ingest locally and run phylo from those local files" to be treated as a first class citizen alongside "ingest, upload to S3, download from S3, run phylo". The use cases are numerous in my opinion, including pipeline development, custom ingests and full runs in GitHub Actions / AWS using the local file system. (This is related to a longstanding desire of mine to sort out multiple inputs, which is a perennial to-do.) 

I'm ambivalent as to the exact approach we use, but I do think a snakemake solution is preferable.

However let's not solve this problem in this PR - happy to discuss further with you & Jover!

Reply:
> However let's not solve this problem in this PR - happy to discuss further with you & Jover!

☝️ Happy to chat about this outside of this PR!